lib/utils.js

```diff
@@ -64,7 +64,11 @@ function getHookValue(data, job, key) {
 }
 
 function makeHook(format, job) {
-  format = format || DEFAULT_FORMAT
+  try {
+    format = JSON.parse(format)
```

Comment:
It wasn't working. `format` always was a string and if you try set your custom payload you'll get an object in which all keys and values are single letters.